### PR TITLE
Added support for rabbitmq vhosts

### PR DESCRIPTION
**Pull request by tomaj**
Added new config option to provide rabbitmq *vhost*.

### CODE REVIEW

File: recipes/rabbit.php [hunk 1]
```diff
@@ -26,12 +26,13 @@
         $config['message'] = "Deployment to '{$host}' on *{$prod}* was successful\n($releasePath)";
     }
 
-    $defaultConfig = array(
+    $defaultConfig = [
```
Comment (gordalina) [inline recipes/rabbit.php]: Could you use PHP 5.3 syntax? We want to have recipes compatible with deployer 2.x

Comment (tomaj) [inline recipes/rabbit.php]: Done

**Comment by gordalina** [comment]
Thanks :+1:

**Comment by tomaj** [comment]
Are you going to release new version in future?

**Comment by gordalina** [comment]
Tagged version 1.0.2

**Pull request merged**